analysis/report/panel_plots.py

```diff
@@ -129,6 +130,115 @@ def reorder_labels(labels):
         )
     return zip(*labels)
 
+def add_percentiles(df, period_column=None, column=None, show_outer_percentiles=True):
+    """For each period in `period_column`, compute percentiles across that
+    range.
+
+    Adds `percentile` column.
+
+    """
+    deciles = numpy.arange(0.1, 1, 0.1)
+    bottom_percentiles = numpy.arange(0.01, 0.1, 0.01)
+    top_percentiles = numpy.arange(0.91, 1, 0.01)
+    if show_outer_percentiles:
+        quantiles = numpy.concatenate((deciles, bottom_percentiles, top_percentiles))
+    else:
+        quantiles = deciles
+    df = df.groupby(period_column)[column].quantile(quantiles).reset_index()
+    df = df.rename(index=str, columns={"level_1": "percentile"})
+    # create integer range of percentiles
+    df["percentile"] = df["percentile"].apply(lambda x: x * 100)
```

Comment:
By removing the int(x) call (to avoid the rounding issue) we end up with a deciles table with the following:
```
2022-09-01,6.000000000000001,0.0,30
2022-09-01,6.999999999999999,0.0,30
```
Functionally this should be fine for the charts, but could be confusing for a reader of the deciles tables

As an alternative, we could call round instead?
```suggestion
    df["percentile"] = df["percentile"].apply(lambda x: round(x * 100))
```